hashring/hashring_test.go

```diff
@@ -310,7 +310,8 @@ func TestLookupNLoopAround(t *testing.T) {
 	ring.AddMembers(members...)
 
 	unique := make(map[valuetype]struct{})
-	ring.tree.LookupNUniqueAt(1, replicaPoint{hash: 0}, unique)
+	orderedUnique := make([]valuetype, 0, 1)
+	ring.tree.LookupNUniqueAt(1, replicaPoint{hash: 0}, unique, &orderedUnique)
```

Comment:
This doesn't test the correctness of the code you've added; please add unit tests to cover the new logic.